S/StrTables/Compat.toml

```diff
@@ -7,5 +7,8 @@ julia = "0.6-1.3"
 ["0.3"]
 julia = "1.0-1.3"
 
-["1.0"]
+["1.0.0"]
 julia = ["1.0.0-1"]
+
+["1.0.1"]
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.